Makefile

```diff
@@ -3,6 +3,8 @@
 help: ##@other Show this help
 	@perl -e '$(HELP_FUN)' $(MAKEFILE_LIST)
 
+GO_VERSION = 0.15.0
```

Comment:
Maybe we can read this from the platform-specific file (.pom file for iOS, `modules/react-native-status/android/build.gradle` for Android)?

Reply:
Yeah, I was thinking about that. Kinda flimsy, but better than hardcoding it.

Reply:
I've dropped `xml.pom`, so now it's only in two places.